### PR TITLE
chore: switch to new dependency updater

### PR DESCRIPTION
**Pull request by vaind**
... that merges changelog until a release

#skip-changelog

### CODE REVIEW

File: .github/workflows/update-deps.yml [hunk 1]
```diff
@@ -11,31 +11,31 @@ on:
 
 jobs:
   cocoa:
-    uses: getsentry/github-workflows/.github/workflows/submodule-updater.yml@923a6414d7e37c07fdd743b2a6ccb40198ead7d3
```
Comment (bitsandfoxes) [inline .github/workflows/update-deps.yml:14]: Do you have plans of switching to tags eventually? Instead of the sha?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,16 +8,16 @@
 - Reduced automated screenshot attachment controls to a simple toggle ([#784](https://github.com/getsentry/sentry-unity/pull/784))
 - Disable AutoSessionTracking on unknown platforms ([#840](https://github.com/getsentry/sentry-unity/pull/840))
 - Support Android apps minified with Proguard ([#844](https://github.com/getsentry/sentry-unity/pull/844))
-- Bump Cocoa SDK to v7.17.0 ([#802](https://github.com/getsentry/sentry-unity/pull/802) & [#821](https://github.com/getsentry/sentry-unity/pull/821))
+- Bump Cocoa SDK to v7.17.0 ([#802](https://github.com/getsentry/sentry-unity/pull/802), [#821](https://github.com/getsentry/sentry-unity/pull/821))
```
Comment (bitsandfoxes) [inline CHANGELOG.md:11]: Neat! I like the `,` better too.

**Review by bitsandfoxes — APPROVED** [review]
LGTM! 🔥

File: .github/workflows/update-deps.yml (hunk 1 above)
Comment (vaind) [reply .github/workflows/update-deps.yml:14]: internal repo so we could, but I just didn't bother creating any yet

**Pull request merged**